app/src/org/commcare/network/HttpRequestEndpointsMock.java

```diff
@@ -1,4 +1,4 @@
-package org.commcare.android.mocks;
+package org.commcare.network;
```

Comment:
Err, maybe it is okay to leave it in... Since you are also using `HttpResponseMock`